### DISCUSSION TITLE
Email Service Supports Unencrypted Authentication

### DISCUSSION THREAD

**Discussion by jshapiro2** [postalserver/postal#3190] [Development discussions]
I have now run security scans on multiple versions of POSTAL including version 3.3.4. I have done all changes to Caddyfile and postal.yml postal to enforce this.  Need postal to stop passing back "PLAIN". 

Here is the issue
SMTP Unencrypted Cleartext Login
CVSS Rating 4.8 (v2)

The remote SMTP server accepts logins via the following cleartext authentication mechanisms over unencrypted connections:

PLAIN
LOGIN

The remote SMTP server supports the 'STARTTLS' command but isn't enforcing the use of it for the cleartext authentication mechanisms.

**Comment by willpower232** [comment]
Unfortunately I don't believe you can turn that off currently but there is nothing stopping you from changing your own version of Postal.

You could remove this line

https://github.com/postalserver/postal/blob/da90e75036c27482699921613d838f4058a100e7/app/lib/smtp_server/client.rb#L98

and edit this line

https://github.com/postalserver/postal/blob/da90e75036c27482699921613d838f4058a100e7/app/lib/smtp_server/client.rb#L159
